.github/workflows/build-gtk.yml

```diff
@@ -38,6 +38,13 @@ jobs:
             dotnet build -c Release Mali.slnf
       - name: Pack MAUI
         run: dotnet pack Mali.slnf
+      - name: Check package versions
+        run: |
+          version_prefix=$(cat GitInfo.txt | tr -d '\n' | cut -d'-' -f1)
+          # fail if at least one of *.nupkg files' names' version portion doesn't start with version string stored in GitInfo.txt
+          # ( and ) have to be escaped when used in find command regex
+          count=$(find . -type f -name "*.nupkg" ! -regex "\(.*\/\)*\([a-zA-Z]+\.\)+${version_prefix}.*" | wc -l)
+          if [ "$count" -gt 0 ]; then exit 1; fi
```

Comment:
@webwarrior-ws this would be VERY CONFUSING for the developer that faces it because it doesn't have an error message

Reply:
added a message